cmd/main.go

```diff
@@ -895,3 +914,31 @@ func startPPROF(log *logrus.Logger) {
 		}
 	}
 }
+
+func runOpenshiftReleaseSyncerIfNeeded(
+	releaseSources models.ReleaseSources,
+	releaseImages models.ReleaseImages,
+	db *gorm.DB,
+	log logrus.FieldLogger,
+	releaseHandler oc.Release,
+) *thread.Thread {
+	if !Options.EnableKubeAPI && Options.ReleaseSourcesConfig.ReleaseSources != "" {
+		releaseSourcesHandler, err := releasesources.NewReleaseSourcesHandler(releaseSources, releaseImages, log, db, Options.ReleaseSourcesConfig, releaseHandler)
+		if err != nil {
+			log.WithError(err).Debug("error occurred while creating release sources handler")
+		}
+		go releaseSourcesHandler.SyncReleaseImages()
```

Comment:
Can you describe why you are starting go routine and then running thread? Why running thread is not enough ?

Reply:
The way `thread` package currently behaving is running the first time after 1 interval. Therefore I ran it once before because I wanted to have the release images in the DB when assisted-service starts

Reply:
If you want it there when the service starts then don't do a go routine.

Do we know how long this takes?